### ISSUE TITLE
LFW Results not reproduced using given EUM weights and Resnet100 pre-trained weights.

### ISSUE THREAD

**Issue by aasharib** [fdbtrs/Self-restrained-Triplet-Loss#5]
I am trying to reproduce the results using given EUM weights, Resnet100 pre-trained weights downloaded from https://github.com/deepinsight/insightface/tree/master/model_zoo and LFW mask type and colors given in LFW-log.txt. But my results are not as good as reported results.
Please provide the link of pre-trained model if you are using different model. Also provide the link of Resnet-50 and MobileNet weights links.

**Comment by moeedabdul** [comment]
Dear @aasharib can you please share the EUM weights with me on moeedabdul528@gmail.com. I can't find them thanks.

**Comment by noameshed** [comment]
I have also tried to reproduce results using given EUM weights [here](https://github.com/fdbtrs/Self-restrained-Triplet-Loss/tree/master/weights/weightsResNet100/SRT) and the Resnet-100 pretrained weights from https://github.com/deepinsight/insightface/tree/master/model_zoo with poor results. Please provide pre-trained models for Resnet-50, Resnet-100, and MobileFaceNet. Thank you!

**Comment by fdbtrs** [comment]
Only masked embedding should be processed by EUM model. you should get exact results as in the paper

**Comment by xueyedamo521** [comment]
Dear, can you provide the feature extraction pretrained weights (Resnet-50, Resnet-100, and MobileFaceNet)? How can i download them?  I cannot reproduce the results in your paper.
